source/mixin/network_mixin.py

```diff
@@ -0,0 +1,40 @@
+from PyQt4.QtCore import *
+from PyQt4.QtNetwork import *
+
+
+class NetworkMixin(QNetworkAccessManager):
```

Comment:
Every class should have  a doctoring. See https://github.com/inasafe/inasafe/wiki/Coding-Standards#doc-strings-and-comments

Also whenever adding a new class or method you should annotate it with the .. versionadded:: sphinx note - see https://github.com/inasafe/inasafe/wiki/Coding-Standards#annotating-api-changes-and-additions